client/extensions/woocommerce/woocommerce-services/views/label-settings/label-settings.js

```diff
@@ -201,11 +201,15 @@ class ShippingLabels extends Component {
 
 			let summary;
 
-			if ( selectedPaymentMethod ) {
-				const { card_type: selectedType, card_digits: selectedDigits } = find( paymentMethods, {
+			const method =
+				selectedPaymentMethod &&
+				find( paymentMethods, {
 					payment_method_id: selectedPaymentMethod,
 				} );
 
+			if ( method ) {
```

Comment:
Looks like a good change, but appears to be out of context – does this fix a particular issue?

Reply:
I am not sure what I did with my packages, but I was constantly getting an "Cannot read `card_type` of `undefined`" error and it would've been too complicated to do this in another PR.